airgun/utils.py

```diff
@@ -69,3 +73,22 @@ def get_widget_by_name(widget_root, widget_name):
                 )
         widget = getattr(widget, name)
     return widget
+
+
+def retry_navigation(method):
+    """Decorator to invoke method one or more times, if TimedOutError is raised."""
+
+    @functools.wraps(method)
+    def retry_wrapper(*args, **kwargs):
+        attempts = 3
+        for i in range(attempts):
+            try:
+                return method(*args, **kwargs)
+            except (Exception):
```

Comment:
Dont you think Exception here is `too open` and it's not very limited to TimeoutError? 

This will end up re-executing the tests that are not failing with `TimeoutError` and taking more time to finish after error!

Reply:
I agree, in any case test should conform to description (and please check description, is it TimedOutError or TimeoutError? I think the latter.)